examples/client/client.c

```diff
@@ -1175,6 +1249,12 @@ static const char* client_usage_msg[][70] = {
 #endif
         "For simpler wolfSSL TLS client examples, visit\n"
         "https://github.com/wolfSSL/wolfssl-examples/tree/master/tls\n", /* 70 */
+#ifdef HAVE_LIBOQS
```

Comment:
Can you move up the new `--ops` help? I'd like the above message about examples to be the last item. Same for 1458 and in server.c

Reply:
Sure.  Will get that done today!!